### PR TITLE
🌐 Add Japanese translation for Help FastAPI

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates the help-fastapi.md to Japanese.
Please review help-fastapi.md

~Issue #1571 generates a new directory to support Japanese.
I will fix dependency after #1571 is merged.~

### CODE REVIEW

File: docs/ja/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* <a href="https://twitter.com/tiangolo" class="external-link" target="_blank">**Twitter** でフォロー</a>。
+    * FastAPIの使用用途を教えてください (聞いてみたいです)。
+    * 新たなツールの発表やリリースが聞けます。
+* <a href="https://www.linkedin.com/in/tiangolo/" class="external-link" target="_blank">**Linkedin** でつながる</a>。
+    * 新たなツールの発表やリリースが聞けます (ただしTwitterの方が利用頻度が高いですが 🤷‍♂)。
+* <a href="https://dev.to/tiangolo" class="external-link" target="_blank">**Dev.to**</a> や <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">**Medium**</a> で著作物を読む (またはフォロー)。
+    * アイデアや作成ツールについての記事が読めます。
+    * 新規記事の執筆を通知してくれます。
+
+## Tweet about **FastAPI**
```
Comment (SwftAlpc) [inline docs/ja/docs/help-fastapi.md:45]: 消し忘れでしょうか？

File: docs/ja/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+## **FastAPI** に関するツイート
+
+<a href="https://twitter.com/compose/tweet?text=I'm loving FastAPI because... https://github.com/tiangolo/fastapi cc @tiangolo" class="external-link" target="_blank">**FastAPI** についてツイート</a>し、開発者や他の人にどこが気に入ったのか教えてください。🎉
+
+**FastAPI** がどのように使われ、どこが気に入られ、どんなプロジェクト/会社で使われているかなどについて知りたいです。
+
+## Vote for FastAPI
```
Comment (SwftAlpc) [inline docs/ja/docs/help-fastapi.md:53]: こちらも消し忘れ

File: docs/ja/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+* 質問、または、問題の報告
+* 新機能の提案
+
+**Note**: issueを立てた人は、他の人の手助けもお願いします。😉
+
+## プルリクエストをする
+
+以下の様な<a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">プルリクエストを作成</a>できます:
+
+* ドキュメントのタイポを訂正。
```
Comment (SwftAlpc) [inline docs/ja/docs/help-fastapi.md:86]: ドキュメントのタイプミスを修正

**Review by SwftAlpc — CHANGES_REQUESTED** [review]
I think it's a very good translation, but you forgot to erase a few sentences, so if you could fix that it would be perfect!

**Review by SwftAlpc — APPROVED** [review]
Thank you!

LGTM

File: docs/ja/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+## FastAPIに投票
+
+* <a href="https://www.slant.co/options/34241/~fastapi-review" class="external-link" target="_blank">Slantで **FastAPI** に投票</a>
+* <a href="https://alternativeto.net/software/fastapi/" class="external-link" target="_blank">AlternativeToで **FastAPI** に投票</a>
+* <a href="https://github.com/marmelab/awesome-rest/pull/93" class="external-link" target="_blank">awesome-restで **FastAPI** に投票</a>
+
+## GitHub issuesで他の人を助ける
+
+<a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">既存のissues</a>を確認して、他の人を助けて見てください。皆さんが回答を知っているかもしれないような質問がほとんどです。🤓
```
Comment (komtaki) [inline docs/ja/docs/help-fastapi.md:57]:
```suggestion
<a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">既存のissues</a>を確認して、他の人を助けてみてください。皆さんが回答を知っているかもしれない質問がほとんどです。🤓
```

微妙ですが、「見」は変換しない方が意味があってる気がします。 🤔 
「ような」はない方が読みやすいかもしれません。

File: docs/ja/docs/help-fastapi.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    <img src="https://badges.gitter.im/tiangolo/fastapi.svg" alt="Join the chat at https://gitter.im/tiangolo/fastapi">
+</a>
+
+Gitterでチャットに参加: <a href="https://gitter.im/tiangolo/fastapi" class="external-link" target="_blank">https://gitter.im/tiangolo/fastapi</a>.
+
+そこで、他の人と手早く会話したり、手助けやアイデアの共有などができます。
+
+しかし、「自由な会話」が許容されているので一般的すぎて回答が難しい質問もしやすくなります。そのせいで回答を得られないかもしれません。
+
+GitHub issuesではテンプレートが良い回答を得やすいような質問ができるように、もしくは、質問する前に自身で解決できるようにガイドしてくれます。そして、GitHubではたとえ時間がかかっても全てに答えているか確認できます。個人的にはGitterチャットでは同じことはできないです。😅
```
Comment (komtaki) [inline docs/ja/docs/help-fastapi.md:97]:
```suggestion
GitHub issuesでは良い回答を得やすい質問ができるように、もしくは、質問する前に自身で解決できるようにテンプレートがガイドしてくれます。そして、GitHubではたとえ時間がかかっても全てに答えているか確認できます。個人的にはGitterチャットでは同じことはできないです。😅
```

語順の入れかえ

File: docs/ja/docs/help-fastapi.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+<a href="https://github.com/sponsors/tiangolo" class="external-link" target="_blank">GitHub sponsors</a>を通して開発者を経済的にサポートできます。
+
+そこで、感謝の気持ちを伝えるためにコーヒー☕️を買うことができます 😄。
+
+## FastAPIを強化するツールのスポンサーになる
+
+ドキュメントで見たように、FastAPIはStarletteとPydanticという巨人の肩に乗っています。
+
+以下のスポンサーなることもできます:
```
Comment (komtaki) [inline docs/ja/docs/help-fastapi.md:113]: ### 
```suggestion
以下のスポンサーになることもできます:
```

タイポです。

**Review by komtaki — COMMENTED** [review]
Thanks for your translate. 👍 
I commented to make it read easier as Japanese.

**Comment by tokusumi** [comment]
@komtaki 
Thank you!

**Comment by komtaki** [comment]
LGTM 🙆‍♂️

**Comment by tokusumi** [comment]
@komtaki  could you approve this?

**Review by komtaki — APPROVED** [review]

**Comment by tokusumi** [comment]
Hi, @tiangolo
This PR has been approved by two reviewers (github actions to label approved PR didn't work, though).
Would you like to require something more for this to be merged?

**Comment by tiangolo** [comment]
Awesome! Thanks @tokusumi ! :coffee: :rocket: 

And thanks @SwftAlpc and @komtaki for the reviews :bow: :nerd_face:

**Pull request merged**